src/node_crypto.cc

```diff
@@ -3261,7 +3261,7 @@ void CipherBase::InitIv(const char* cipher_type,
      iv_length (0) for ECB ciphers */
   if (EVP_CIPHER_iv_length(cipher_) != iv_len &&
       !(EVP_CIPHER_mode(cipher_) == EVP_CIPH_ECB_MODE && iv_len == 0) &&
```

Comment:
Seeing the above comment in regard of the openssl bug, the issue of ECB mode has already resolved in    https://github.com/openssl/openssl/commit/97fe2b40c1f39fd39885862f9b4da64bf587ed20 so that we no longer need to check the iv length of ECB mode.